media-gfx/openvdb/files/openvdb-6.2.1-fix-multilib-header-source.patch

```diff
@@ -0,0 +1,29 @@
+diff -Naur a/cmake/FindIlmBase.cmake b/cmake/FindIlmBase.cmake
```

Comment:
See patch comments above.

Reply:
These patches all solve the same issue, allowing cmake to determine the installed version of ilmbase and openexr, in different ways. The original patch by dracwyrm was for 4.0.2, and was updated for 5.2.0 to account for some changes in the surrounding context which prevented it from being applied.

In 6.2.1 and later the issue is solved based on a patch by Bernd Waibel which changes the directory that cmake searches for the header files so that it can parse it for the major and minor version numbers. In multilib they are placed under /usr/include/${CHOST} rather than /usr/include which only contains a placeholder.